r/README.md

```diff
@@ -178,6 +178,28 @@ For any other build/configuration challenges, see the [C++ developer
 guide](https://arrow.apache.org/docs/developers/cpp/building.html) and
 `vignette("install", package = "arrow")`.
 
+### Using `remotes::install_github(...)` 
+
+If you need an Arrow installation from a specific repository or at a specific ref, `remotes::install_github()` should work on most platforms. This method is helpful if you need a full install of arrow that is separate from another install (e.g. we use this in [arrowbench](https://github.com/ursacomputing/arrowbench) to install development versions of arrow isolated from the system install). However there are some caveats to be aware of:
```

Comment:
> `remotes::install_github()` should work on most platforms

I'm not sure how true that is. It's not true on Windows. 

Either way, we definitely don't want to recommend it to users--we'd rather they install nightly packages if they want to test the latest. Since we want to deemphasize this, and since the concerns you address below (where you already have a C++ library built separately and installed on your system) are things that only developers would face, I think we should move this, perhaps to the install.Rmd vignette for now. We should revise the readme and vignettes more holistically before the release as well; Ian made a JIRA about that a while back.

Reply:
Yeah, absolutely. I'll move all of this over to the install vignette. I agree about a more holistic re-org, but going to punt on _that_ for now